Makefile.PL

```diff
@@ -26,11 +26,13 @@ requires        ('List::Util');
 requires        ('Encode');
 requires        ('File::Find');
 requires        ('IO::Socket::INET');
+requires        ('File::Copy::Recursive');
```

Comment:
@spacewander Test::Nginx currently heavily uses fork and signals which won't work in native win32 environment without UNIX-like emulation (like Cygwin) and in the latter case we do have `cp` anyway.